content/docs/command-reference/version.md

```diff
@@ -36,36 +34,36 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: official releases (for example `1.7.2`) are stable
+  versions that have upgraded features and bug fixes.
+  [These instructions](/doc/install) include ways to install DVC using the
+  official package stored in [Python Packaging Authority](https://www.pypa.io/).
+  We mark these official releases with
+  [tags](https://github.com/iterative/dvc/releases) on DVC's repository. Any
```

Comment:
How about `Releases are [tagged](...) in the DVC repository.` ? Simple, direct.

Reply:
p.s. we could take the opportunity to mention that's our changelog, or use the term somehow in the sentence.